andela/ah-the-unsullied-frontend
# #162949189 Bookmark Article

#### What does this PR do?
- Bookmark articles
#### Description of Task to be completed?
- Only an authenticated user can bookmark an article
#### How should this be manually tested?
1. Clone the repository
```
git clone https://github.com/andela/ah-the-unsullied-frontend.git
```
2. Cd into the cloned repo
```
cd ah-the-unsullied-frontend
```
3. Checkout to this branch
```
git checkout ft-tags-162949181 
```
4. Install the dependencies of the project
```
npm install
```
5. Run the app
```
npm start
```
6. Visit the default route `http://localhost:3000/` 

7. Create an account and verify it from the email

8. Log into your account and on the landing page click on an article

9. On the article view page click on the bookmark icon just above the social site's icons to bookmark

#### Any background context you want to provide?
You have to be an authenticated user to bookmark an article
#### What are the relevant pivotal tracker stories?
#162949189 
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/28457081/53636860-a6718680-3c32-11e9-822a-7175d6786fa6.png)

#### Questions:
